### PR TITLE
:memo: added lozad.js in tools

### PR DESCRIPTION
**Pull request by ApoorvSaxena**
added lozad.js in tools section

### CODE REVIEW

File: data/tools/lozad.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name" : "lozad.js",
+  "javascript" : {
+    "url" : "https://github.com/ApoorvSaxena/lozad.js"
+  },
+  "description" : "Performant lazy loader using IntersectionObserver API",
+  "tags" : [ "loading", "lazy loading", "images", "iframes" ]
+}
```
Comment (stefanjudis) [inline data/tools/lozad.json]: Hmm. Why the `iframes` tag?

Comment (ApoorvSaxena) [reply data/tools/lozad.json]: it allows lazy loading for all sort of elements be images, iframes, videos etc. and considering iframes is a very common use case thus have added in the tag

Comment (stefanjudis) [reply data/tools/lozad.json]: Ja, that's what I thought. `images` works for me as a common use case, but I wouldn't add a new for `iframes`.

I'm happy to take the PR without this fourth tag. Does this work for you?

**Comment by ApoorvSaxena** [comment]
@stefanjudis fixed as per your comment

**Comment by stefanjudis** [comment]
👍

**Pull request merged**